src/main/scala/util/GeneratorUtils.scala

```diff
@@ -111,7 +113,11 @@ trait GeneratorApp extends App with HasGeneratorUtilities {
   def generateAnno {
     val annotationFile = new File(td, s"$longName.anno")
     val af = new FileWriter(annotationFile)
-    af.write(circuit.annotations.toArray.toYaml.prettyPrint)
+    val moduleName  = ModuleName(circuit.name, CircuitName(circuit.name))
+    println(s"Target dir: $td/")
```

Comment:
Rocket puts stuff in `generated-src`, without this the black box source transform will put it in the directory where you invoke make. Right now, different rocket configurations will have black boxes clobber each other, which seems bad. That's the current status quo, though, so maybe not a big deal.